packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -39,6 +39,7 @@ class ExpansionTile extends StatefulWidget {
     this.onExpansionChanged,
     this.children: const <Widget>[],
     this.trailing,
+    this.initiallyExpanded: false,
   }) : super(key: key);
```

Comment:
assert that initiallyExpanded is not null, and mention in the constructor docs that it must not be null. (See some other widgets for the pattern to use.)